plugins/automox/setup.py

```diff
@@ -3,7 +3,7 @@
 
 
 setup(name="automox-automox-plugin",
-      version="2.0.0",
+      version="2.0.1",
```

Comment:
should there be a changelog in help.md for this as well?

Reply:
Looks like there is items up to v2.0.0 https://github.com/rapid7/insightconnect-plugins/blob/master/plugins/automox/help.md#version-history

Reply:
I looked in the wrong place... In develop there's a record for `2.0.1` https://github.com/rapid7/insightconnect-plugins/blob/develop/plugins/automox/help.md?plain=1#L1496

Reply:
My bad! 🤦